packages/core/src/app/TimeEstimator.ts

```diff
@@ -67,6 +73,10 @@ export class TimeEstimator {
     this.lastUpdateTimestamp = timestamp;
   }
 
+  public reportProgress() {
+    window.logProgress(window.location.port, this.completion.current);
```

Comment:
You can use `(window as any).logProgress`, that way you don't have to declare any global functions.